content/_partials/books.adoc

```diff
@@ -0,0 +1,6 @@
+[NOTE]
+====
+The books in this list are not endorsed by the Jenkins project.
+The list of books on this page is provided for informational purposes, sorted from newest to oldest publication date.
+Since older books on the list can contain outdated information, the link:/book/getting-started/[Jenkins documentation] can  help fill in any gaps.
```

Comment:
```suggestion
Since older books on the list can contain outdated information, the link:/doc/[Jenkins documentation] can help fill in any gaps.
```
There's no getting started on the books page, do you mean this page, by any chance?

Reply:
@NotMyFault, yes that is what I had intended to put there, I removed the /doc/ part instead of the /book/ part from the url I was using. Thanks for catching that!!